docs/content/en/docs/reference/k8ssandra/_generated.md

```diff
@@ -38,14 +38,12 @@
 | cassandra.cassandraLibDirVolume.storageClass | string | `"standard"` | Storage class for persistent volume claims (PVCs) used by the underlying cassandra pods. Depending on your Kubernetes distribution this may be named "standard", "hostpath", or "localpath". Run `kubectl get storageclass` to identify what is available in your environment. |
 | cassandra.cassandraLibDirVolume.size | string | `"5Gi"` | Size of the provisioned persistent volume per node. It is recommended to keep the total amount of data per node to approximately 1 TB. With room for compactions this value should max out at ~2 TB. This recommendation is highly dependent on data model and compaction strategies in use. Consider testing with your data model to find an optimal value for your usecase.  |
 | cassandra.allowMultipleNodesPerWorker | bool | `false` | Permits running multiple Cassandra pods per Kubernetes worker. If enabled resources.limits and resources.requests **must** be defined. |
-| cassandra.heap | object | `{"newGenSize":null,"size":null}` | Cluster-level heap configuration |
-| cassandra.resources | object | `{}` | Resource requests for each Cassandra pod. |
+| cassandra.resources | object | `{}` | Cluster-level heap configuration (optional). See reference: https://k8ssandra.io/docs/reference/k8ssandra/#values for additional information. size: newGenSize: -- Resource requests for each Cassandra pod. |
```

Comment:
Similar to above this is out of place.

Reply:
Right, this goes to @jakerobb's [comment](https://github.com/k8ssandra/k8ssandra/pull/340#discussion_r574079468) about the comment getting merged into the next field. Removing the `--` fixes the issue, but the comment is not rendered in the docs (which may be what we want).